docs/metrics.md

```diff
@@ -0,0 +1,166 @@
+| prometheus_tsdb_compaction_chunk_size_bytes | histogram  | Final size of chunks on their first compaction.                                                                                                                        |
+| prometheus_tsdb_compaction_duration_seconds | histogram  | Duration of compaction runs.                                                                                                                        |
+| prometheus_tsdb_compaction_populating_block | gauge  | Set to 1 when a block is currently being written to the disk.                                                                                                                        |
+| prometheus_tsdb_compactions_failed_total | counter  | Total number of compactions that failed for the partition.                                                                                                                        |
+| prometheus_tsdb_compactions_skipped_total | counter  | Total number of skipped compactions due to disabled auto compaction.                                                                                                                        |
+| prometheus_tsdb_compactions_total | counter  | Total number of compactions that were executed for the partition.                                                                                                                        |
+| prometheus_tsdb_compactions_triggered_total | counter  | Total number of triggered compactions for the partition.                                                                                                                        |
+| prometheus_tsdb_data_replay_duration_seconds | gauge  | Time taken to replay the data on disk.                                                                                                                        |
+| prometheus_tsdb_exemplar_exemplars_appended_total | counter | Total number of appended exemplars.                                                                                                                        |
+| prometheus_tsdb_exemplar_exemplars_in_storage | gauge | Number of exemplars currently in circular storage.                                                                                                                        |
+| prometheus_tsdb_exemplar_last_exemplars_timestamp_seconds | gauge | The timestamp of the oldest exemplar stored in circular storage. Useful to check for what timerange the current exemplar buffer limit allows. This usually means the last timestampfor all exemplars for a typical setup. This is not true though if one of the series timestamp is in future compared to rest series.                                                                                                                        |
```

Comment:
This is a bug in the source; even better if you fix that.

```suggestion
| prometheus_tsdb_exemplar_last_exemplars_timestamp_seconds | gauge | The timestamp of the oldest exemplar stored in circular storage. Useful to check for what time range the current exemplar buffer limit allows. This usually means the last timestamp for all exemplars for a typical setup. This is not true though if one of the series timestamp is in future compared to rest series.                                                                                                                        |
```

Reply:
I have resolved it with the new commit.

Reply:
Thanks; are you going to do the Go source?

Reply:
I have no idea of the Go source metrics; could you expantiate more on it?